### PR TITLE
fix(security): security version updates

### PR DESCRIPTION
**Pull request by david-leifker**
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -184,6 +191,7 @@ configure(subprojects.findAll {! it.name.startsWith('spark-lineage') }) {
 
   configurations.all {
     exclude group: "io.netty", module: "netty"
+    exclude group: "log4j", module: "log4j"
```
Comment (david-leifker) [inline build.gradle:194]: This does seem to do something, what is not covered here is the -core and -api modules, but those are locked at the latest version

File: buildSrc/build.gradle [hunk 2]
```diff
@@ -10,7 +10,7 @@ dependencies {
     exclude group: 'com.google.guava', module: 'guava'
   }
   compile 'com.google.guava:guava:27.0.1-jre'
-  compile 'com.fasterxml.jackson.core:jackson-databind:2.9.10.7'
-  compile 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.8.11'
+  compile 'com.fasterxml.jackson.core:jackson-databind:2.13.4.2'
```
Comment (RyanHolstien) [inline buildSrc/build.gradle:13]: Variable replacement here too

**Review by RyanHolstien — APPROVED** [review]
LGTM :) minor comment for consistency

File: buildSrc/build.gradle (hunk 2 above)
Comment (david-leifker) [reply buildSrc/build.gradle:13]: This can't be done in the `buildSrc` like normal, unless the version config is refactored out of its currently location.

**Pull request merged**